src/sentry/tagstore/snuba/backend.py

```diff
@@ -382,13 +382,17 @@ def get_group_event_filter(self, project_id, group_id, environment_id, tags):
 
         conditions = [[['tags[{}]'.format(k), '=', v] for (k, v) in tags.items()]]
 
-        result = snuba.query(start, end, groupby=['event_id'], conditions=conditions,
-            filter_keys=filters, limit=1000, referrer='tagstore.get_group_event_filter')
+        result = snuba.raw_query(
+            start, end, groupby=None, selected_columns=['event_id'],
+            conditions=conditions, orderby='-timestamp', filter_keys=filters,
```

Comment:
So this kinda relies on some weird behavior that I think we need to clean up first. When `query` and `raw_query` were separated, the argument and result translation was kinda split between them in a weird way. We now get the translators twice, but only do the forward translation in raw_query, and the reverse translation in `query`. 

This will have weird effects in cases where we are querying with eg. environment filters, when we get back either environment names or environment ids depending on if we call `raw_query` or `query`. cc @lynnagara